src/message/renderMessages.js

```diff
@@ -54,23 +39,10 @@ export default (messages: Message[], narrow: Narrow) => {
       prevItem &&
       prevItem.sender_full_name === item.sender_full_name;
 
-    if (isOutbox) {
-      list.push(
-        <OutboxMessageContainer
-          type="outbox"
-          key={item.timestamp}
-          isBrief={shouldGroupWithPrev}
-          message={item}
-        />,
-      );
-      prevItem = item;
-      continue; // eslint-disable-line
-    }
-
     list.push(
       <MessageContainer
-        type="message"
-        key={item.id}
+        type={isOutbox ? 'outbox' : 'message'}
```

Comment:
Same here. The type should be 'message' and outbox-iness should be determined in other way.